assets/js/components/AboutPage/AboutPage.test.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { aboutPageFactory } from '@lib/test-utils/factories';
+import AboutPage from './index';
+import { act, screen } from '@testing-library/react';
+import { renderWithRouter } from '@lib/test-utils';
+
+describe('AboutPage component', () => {
+  const apiRequestData = aboutPageFactory.build();
+  it('should render the about page with content from the api', async () => {
+    await act(async () => {
+      renderWithRouter(
```

Comment:
Usually is a good practice to do something like:
```
{ getByText } = renderWithRouter(..
expect(getByText(api..
```
This way you are 100% sure that the used `getByText` is applied in the rendered component